testing/playwright/fixtures/i18n.ts

```
@@ -0,0 +1,80 @@
+					}
+					// eslint-disable-next-line no-restricted-syntax
+					return Object.keys(
+						window.i18n.store.data[languageInner] ?? {},
+					) as Namespace[];
+				},
+				[language] as const,
+			);
+			return namespaces;
+		});
+	},
+	getI18nResource: async ({}, use) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)